src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -351,3 +414,67 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
     CurrentPower = Power;
     return Power;
 }
+
+#if defined(TARGET_TX_BETAFPV_2400_V1) || defined(TARGET_TX_BETAFPV_900_V1)
```

Comment:
Why not seperate these two targets, so the power of betafpv 900 tx can be adjusted to 1W

Reply:
It can not be adjusted to 1W currently